src/main/java/mods/eln/misc/Obj3D.java

```diff
@@ -518,7 +518,9 @@ public boolean loadFile(final String filePath, boolean reload) {
                 String line;
                 while ((line = bufferedReader.readLine()) != null) {
                     String[] words = line.split(" ");
-                    if (words[0].equals("o")) {
+                    if (words[0].startsWith("#")) {
+                        // # is a comment - ignore line.
```

Comment:
Unknown verbs are already ignored. I guess we should log a debug warning for those?